quiz/proxy/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM nginx:1.23.3
+
+EXPOSE 80
+EXPOSE 443
```

Comment:
Wydaje mi się to niepotrzebne tutaj, porty które są wystawione powinien za to odpowiadać orchestrator jak compose lub kubernetes

Reply:
removed